I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/episode0.rpy", line 258, in script call
    call episode_1
  File "game/episode1.rpy", line 69, in script call
    call freeroam
  File "game/freeroam.rpy", line 6, in script call
    call episode_1_cleared
  File "game/freeroam.rpy", line 18, in script
    textbutton "View Profile"
Exception: Sayer 'textbutton' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/episode0.rpy", line 258, in script call
    call episode_1
  File "game/episode1.rpy", line 69, in script call
    call freeroam
  File "game/freeroam.rpy", line 6, in script call
    call episode_1_cleared
  File "game/freeroam.rpy", line 18, in script
    textbutton "View Profile"
  File "C:\Users\tusha\Downloads\Programs\Renpy\renpy-8.1.1-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\tusha\Downloads\Programs\Renpy\renpy-8.1.1-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'textbutton' is not defined.

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
day1 1.0
Sat Apr 13 12:41:09 2024
